api/src/utils/media/xlsx/xlsx-file.ts

```diff
@@ -33,6 +38,32 @@ export class XLSXCSV {
     return mediaValidation.getState();
   }
 
+  /**
+   * Determines the validity of a workbook belonging to a submission. Workbook validators
+   * are different in that they validate an entire submission, but return validations
+   * particular to worksheets within the workbook.
+   * @param validationSchemaParser
+   * @returns
+   */
+  isWorkbookValid(validationSchemaParser: ValidationSchemaParser): ICsvState[] {
```

Comment:
This JS doc comment pretty much sums up my justification for creating this method, and the added validation schema parser methods that go along with it. But essentially, `isWorkbookValid` is a fringe method that returns `ICsvState[]` for an entire submission, whereas the other two methods return `IMediaState` for an entire submission, or `ICsvState[]` for a single file in a submission.